check_python_format

```diff
@@ -0,0 +1,96 @@
+#! /bin/bash
+
+#
+# A simple script for verifying that all the commits in the PR-range has
+# the code correctly formatted with the Black code formatter.
+#
```

Comment:
Is this true? By reading the script, I think it will verify the given directory (`.` by default) regardless of the PR-range.

Reply:
I think we should check only the PR-range, as you have intended in the docs, so that we don't force huge PRs just formatting (which I personally dislike as they kill `git blame`)